core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -4430,16 +4430,16 @@ class KafkaApisTest extends Logging {
     when(sharePartitionManager.fetchMessages(any(), any(), any(), any())).thenReturn(
       CompletableFuture.completedFuture(Map[TopicIdPartition, ShareFetchResponseData.PartitionData](
         new TopicIdPartition(topicId, new TopicPartition(topicName, partitionIndex)) ->
-                  new ShareFetchResponseData.PartitionData()
-                    .setErrorCode(Errors.NONE.code)
-                    .setAcknowledgeErrorCode(Errors.NONE.code)
-                    .setRecords(records)
-                    .setAcquiredRecords(new util.ArrayList(List(
-                      new ShareFetchResponseData.AcquiredRecords()
-                        .setFirstOffset(0)
-                        .setLastOffset(9)
-                        .setDeliveryCount(1)
-                      ).asJava))
+          new ShareFetchResponseData.PartitionData()
+            .setErrorCode(Errors.NONE.code)
+            .setAcknowledgeErrorCode(Errors.NONE.code)
+            .setRecords(records)
+            .setAcquiredRecords(new util.ArrayList(List(
+              new ShareFetchResponseData.AcquiredRecords()
+                .setFirstOffset(0)
+                .setLastOffset(9)
+                .setDeliveryCount(1)
+            ).asJava))
```

Comment:
Is it an inteded change?

Reply:
Thanks for the review. It wasn't intended, but I guess the current indentation is fine and we can leave it as it is.